frontend/catalyst/utils/temp_env.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2022-2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Temporarily modify environment variables.
+"""
+import os
+
+# pylint: disable=missing-class-docstring
+class TempEnv:
```

Comment:
If this is only used for testing I think it should be moved to the test folder.

Reply:
At the moment, yes. But I can see it useful in other situations. I prefer to keep it in utilities for now.

Reply:
This means we are shipping it to users without a use for it though, right?

Reply:
It also seems responsible for a drop in coverage, putting it the tests folder would exclude it from that.

Reply:
Moved it.

Reply:
[Deleted it.](https://github.com/PennyLaneAI/catalyst/pull/30/commits/9c2975b8bedbdbdd4a81f1865308539c86a48c2c)